manifests/01-cluster-role.yaml

```diff
@@ -0,0 +1,122 @@
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
Meaning this file should be removed and instead modify the ClusterRole in 05_deployment? This might be why my e2e is failing on missing permissions for cloudcredentials resources. 

Pushed a commit to address.